### PR TITLE
Update intune-notices.md

### PR DESCRIPTION
**Pull request by guenthst**
Line 16 state "Q1 2023" but line 20 was still showing "November"

### CODE REVIEW

File: memdocs/intune/includes/intune-notices.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ As we continue to invest in Setup Assistant with modern authentication, which is
 
 ### How does this affect you or your users?
 
-In November, new enrollments (new devices that are targeted with an existing enrollment profile or devices re-enrolling) that are targeted with an existing enrollment profile with the Company Portal authentication method, will not be able to enroll.
+In Q1 2023, new enrollments (new devices that are targeted with an existing enrollment profile or devices re-enrolling) that are targeted with an existing enrollment profile with the Company Portal authentication method, will not be able to enroll.
```
Comment (JohanFreelancer9) [inline memdocs/intune/includes/intune-notices.md:20]:
```suggestion
As of the first quarter of 2023, new enrollments (new devices that are targeted with an existing enrollment profile or devices re-enrolling) that are targeted with an existing enrollment profile with the Company Portal authentication method, will not be able to enroll.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
@dougeby Can you reassign this file to an Intune writer? Thanks!

**Comment by yogkumgit** [comment]
@dougeby Kindly update. Thanks

**Comment by dougeby** [comment]
#sign-off

**Pull request merged**